storage/remote/codec.go

```diff
@@ -158,7 +158,10 @@ func FromQueryResult(res *prompb.QueryResult) storage.SeriesSet {
 			samples: ts.Samples,
 		})
 	}
-	sort.Sort(byLabel(series))
+
+	if sortSeries {
```

Comment:
Is there ever a case where we don't want this sorted?

Reply:
We query JUST one remote read storage? (based on required matchers? :thinking:

Reply:
I think we'd still try to query local storage then.